core/robstoll-lib/atrium-core-robstoll-lib-jvm/src/main/kotlin/ch/tutteli/atrium/core/robstoll/lib/reporting/DetailedObjectFormatter.kt

```diff
@@ -44,7 +44,7 @@ actual class DetailedObjectFormatter actual constructor(
      * - [StringBasedRawString] is represented as [StringBasedRawString.string]
      * - [TranslatableBasedRawString] is represented as result of its translation (by [translator])
      * - [Class] is represented as "[Class.getSimpleName] ([Class.getName])"
-     * - [KClass] is represented as "[Class.getSimpleName] ([Class.getName])"
+     * - [KClass] is represented as "[Class.getName]"
```

Comment:
That's not correct but the previous is neither. Change it to:
```suggestion
     * - [KClass] is represented as "[KClass.simpleName] ([KClass.qualifiedName])" unless the [KClass.qualifiedName] 
     *   differs from [Class.getName] in which case, "-- Class: [Class.getName]" is appended in addition
```